src/Setup/WixBA/WixBA.csproj

```diff
@@ -1,19 +1,48 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information. -->
-
-
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="12.0">
   <PropertyGroup>
     <ProjectGuid>{7C27518B-84AD-4679-8EF4-29DF552CF1AC}</ProjectGuid>
     <AssemblyName>WixBA</AssemblyName>
     <OutputType>Library</OutputType>
     <RootNamespace>WixToolset.UX</RootNamespace>
   </PropertyGroup>
-
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|Win32'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|arm'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x86'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|Win32'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|arm'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x64'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x86'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
```

Comment:
OK

On Mon, Aug 1, 2016 at 12:07 AM, Sean Hall <notifications@github.com> wrote:

> In src/Setup/WixBA/WixBA.csproj
> <https://github.com/wixtoolset/wix4/pull/208#discussion_r72924638>:
>
> > +  </PropertyGroup>
> > +  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'">
> > +    <Prefer32Bit>false</Prefer32Bit>
> > +  </PropertyGroup>
> > +  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|Win32'">
> > +    <Prefer32Bit>false</Prefer32Bit>
> > +  </PropertyGroup>
> > +  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|arm'">
> > +    <Prefer32Bit>false</Prefer32Bit>
> > +  </PropertyGroup>
> > +  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x64'">
> > +    <Prefer32Bit>false</Prefer32Bit>
> > +  </PropertyGroup>
> > +  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x86'">
> > +    <Prefer32Bit>false</Prefer32Bit>
> > +  </PropertyGroup>
>
> These need to be deleted. Prefer32Bit doesn't mean anything for DLLs.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/wixtoolset/wix4/pull/208/files/28721ab8e4985a5641b3efd7975a67cecd8f8d4a#r72924638>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHyCLSbSisJ5rcGTI4Wjs3kUXQfTpV9uks5qbX8igaJpZM4JVm8P>
> .
>